src/Libraries/CoreNodeModels/HigherOrder/MapCombineLacing.cs

```diff
@@ -263,11 +263,11 @@ public class Reduce : VariableInputNode
 
         public Reduce()
         {
-            InPortData.Add(new PortData("reductor", Resources.ReducePortDataReductorToolTip));
-            InPortData.Add(new PortData("seed", Resources.ReducePortDataSeedToolTip));
-            InPortData.Add(new PortData("list1", Resources.PortDataListToolTip + " #1"));
+            InPorts.Add(new PortModel(PortType.Input, this, new PortData("reductor", Resources.ReducePortDataReductorToolTip)));
```

Comment:
just curious... what happens when a zero touch node author adds a PortModel of type Output to the `InPorts`collection... do we crash?